environment/commands/prepare.js

```diff
@@ -15,9 +15,9 @@ class PrepareCommand {
         for (const e in process.env) {
             env[e] = process.env[e];
         }
-        env.LOCAL_BRIDGE_SRC = RainbowConfig.getParam('bridge-src');
-        env.LOCAL_CORE_SRC = RainbowConfig.getParam('core-src');
-        env.LOCAL_NEARUP_SRC = RainbowConfig.getParam('nearup-src');
+        env.LOCAL_BRIDGE_SRC = RainbowConfig.getParam('bridge-src') && path.resolve(RainbowConfig.getParam('bridge-src'));
```

Comment:
Should it be just `path.resolve(RainbowConfig.getParam('bridge-src'))`?

Reply:
No that didn't work, because by default:
```
path.resolve(RainbowConfig.getParam('bridge-src'))
=> path.resolve('')
=> absolute path of rainbow-bridge dir
```